app/classes/document_query.rb

```diff
@@ -6,13 +6,21 @@ class DocumentQuery
   HIGHLIGHT_OPTIONS = {
     pre_tags: ["\ue000"],
     post_tags: ["\ue001"]
-  }
+  }.freeze
 
   DEFAULT_STOPWORDS = %w[
     a an and are as at be but by for if in into is it
     no not of on or such that the their then there these
     they this to was will with
-  ]
+  ].freeze
+
+  FACET_FIELDS = %i[audience
```

Comment:
I'm not totally opposed to `FACET_FIELDS`, but I'm thinking `AGGREGATION_FIELDS` might be better for consistency with the Elasticsearch terminology. Your choice.

Reply:
Renamed.